README.md

```diff
@@ -25,17 +25,17 @@ as bundle splitting and `fusion-react` provides tools to do it easily.
 - [Usage](#usage)
```

Comment:
The diff on the README is a mess. Summary:
- Add [DEPRECATED] to ProviderPlugin, Provider, ProvidedHOC
- Moved new methods to the top
- Moved deprecated methods to the bottom
- middleware link was dead, removed
- Added API reference for withServices, mysteriously missing
- Document new token parameter for ProvidedHOC.create
- Replace PluginProvider/HOC example with useService example